client/order/orderbook_test.go

```diff
@@ -456,31 +456,31 @@ func TestOrderBookUnbook(t *testing.T) {
 			),
 			wantErr: false,
 		},
-		{
-			label: "Unbook sell order with outdated sequence value",
-			orderBook: makeOrderBook(
-				2,
-				"ob",
-				[]*Order{
-					makeOrder([32]byte{'b'}, msgjson.SellOrderNum, 10, 1, 2),
-					makeOrder([32]byte{'c'}, msgjson.SellOrderNum, 10, 2, 5),
-				},
-				make([]*cachedOrderNote, 0),
-				true,
-			),
-			note: makeUnbookOrderNote(0, "ob", [32]byte{'a'}),
-			expected: makeOrderBook(
-				2,
-				"ob",
-				[]*Order{
-					makeOrder([32]byte{'b'}, msgjson.SellOrderNum, 10, 1, 2),
-					makeOrder([32]byte{'c'}, msgjson.SellOrderNum, 10, 2, 5),
-				},
-				make([]*cachedOrderNote, 0),
-				true,
-			),
-			wantErr: false,
-		},
+		// {
+		// 	label: "Unbook sell order with outdated sequence value",
+		// 	orderBook: makeOrderBook(
+		// 		2,
+		// 		"ob",
+		// 		[]*Order{
+		// 			makeOrder([32]byte{'b'}, msgjson.SellOrderNum, 10, 1, 2),
+		// 			makeOrder([32]byte{'c'}, msgjson.SellOrderNum, 10, 2, 5),
+		// 		},
+		// 		make([]*cachedOrderNote, 0),
+		// 		true,
+		// 	),
+		// 	note: makeUnbookOrderNote(0, "ob", [32]byte{'a'}),
+		// 	expected: makeOrderBook(
+		// 		2,
+		// 		"ob",
+		// 		[]*Order{
+		// 			makeOrder([32]byte{'b'}, msgjson.SellOrderNum, 10, 1, 2),
+		// 			makeOrder([32]byte{'c'}, msgjson.SellOrderNum, 10, 2, 5),
+		// 		},
+		// 		make([]*cachedOrderNote, 0),
+		// 		true,
+		// 	),
+		// 	wantErr: false,
+		// },
```

Comment:
Did you have plans for the commented tests in this file?

Reply:
Yeah. I may want to re-implement an error return value for sequence errors, but it needs to be hashed out.